### DISCUSSION TITLE
Unable to access copilot service

### DISCUSSION THREAD

**Discussion by AlgoOy** [community/community#116456] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

<img width="710" alt="image" src="https://github.com/community/community/assets/44948890/09025568-22d5-441d-a487-8a70da4b1269">

In "Plans and usage", I saw that I can use copilot, but in copilot, I did not get this service. Even if I clicked "Start free trial", the page just refreshed without any changes.

<img width="719" alt="image" src="https://github.com/community/community/assets/44948890/9252a9f9-f375-4303-985e-891cdb6fe4aa">

Can anyone help me? Thank you so much!

**Comment by davecheney** [comment, marked as answer]
@AlgoOy 
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
